topic-offset-service/src/main/java/uk/gov/gchq/palisade/service/topicoffset/model/TopicOffsetResponse.java

```diff
@@ -26,15 +26,15 @@
 import java.util.StringJoiner;
 
 /**
- * The TopicOffsetResponse is the output for topic-offset-service. It will only be created if the request message was
- * found to be the start of the messages for a specific client request. It will will contain the offset in the message
+ * The TopicOffsetResponse is the output for Topic-Offset Service. It will only be created if the request message was
+ * found to be the start of the messages for a specific client request. It will contain the offset in the message
  * queue for this first message. This is forwarded to the masked-resource-offset stream where it is used to prepare for
  * the client request to retrieve the data.
```

Comment:
```suggestion
 * The TopicOffsetResponse is the output for Topic-Offset Service. It will only be created if the request message was
 * found to be the start marker for the messages of a specific client request. It will contain the offset in the message
 * queue for this first message. This is forwarded to the masked-resource-offset Kafka topic where it is used to prepare for
 * the client request to retrieve the data.
```